.github/workflows/coverage.yaml

```diff
@@ -18,9 +18,7 @@ jobs:
         with:
           toolchain: nightly
           override: true
-      - run: sudo apt-get update
-      # only for arci-gamepad-gilrs
-      - run: sudo apt-get install -y libudev-dev libasound2-dev
+      - run: ci/ubuntu-install-dependencies.sh
       # only for arci-ros test (roscore)
       - run: sudo apt-get install -y python-roslaunch python-rostopic
```

Comment:
The installations of python-roslaunch and python-rostopic don't work well in `ros:melodic-ros-base` images, so they're leaved for now.

https://github.com/openrr/openrr/runs/1977489062?check_suite_focus=true

```text
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 python-roslaunch : Depends: python-roslib but it is not going to be installed
 python-rostopic : Depends: python-rospy but it is not going to be installed
                   Depends: python-rosbag but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

Reply:
I think if you are using ros:melodic-ros-base, they have been already installed. So you can skip it.
(Add some option to skip?)

Reply:
ros:melodoc-ros-base means that `(base of )ROS is (fully) installed`, and I want to test on `NOT ROS installed` here.(but needs a small part of ROS for testing)
I mean ros:melodoc-ros-base should not be used basicly, and keep this as you did.